### ISSUE TITLE
Unfixed vulnerabilities

### ISSUE THREAD

**Issue by larseggert** [taskcluster/taskgraph#530]
Pulling in Taskcluster via https://github.com/mozilla/neqo/pull/1935 made neqo's [OSSF score](https://securityscorecards.dev/viewer/?uri=github.com/mozilla/neqo) drop because of unpatched vulnerabilities in taskcluster:

Reason
7 existing vulnerabilities detected
Details
Warn: Project is vulnerable to: GHSA-jjg7-2v4v-x38h
Warn: Project is vulnerable to: GHSA-h5c8-rqwp-cp95
Warn: Project is vulnerable to: GHSA-h75v-3vvj-5mfj
Warn: Project is vulnerable to: GHSA-9wx4-h78v-vm56
Warn: Project is vulnerable to: GHSA-34jh-p97f-mpxf
Warn: Project is vulnerable to: GHSA-g4mx-q9vg-27p4 / PYSEC-2023-212
Warn: Project is vulnerable to: GHSA-v845-jxx5-vc9f / PYSEC-2023-192

**Comment by jcristau** [comment]
These are not vulnerabilities in taskcluster.  The report is also older than #1935 getting merged, as far as I can tell.

**Comment by larseggert** [comment]
Yeah, no. The file `.taskcluster/requirements.txt` was created by `taskgraph init`; we made no modifications.

**Comment by jcristau** [comment]
mozilla/neqo#1935 deletes that file...

**Comment by larseggert** [comment]
No, it moves it to `taskcluster` without changes.

**Comment by larseggert** [comment]
You're right. The OSSF score was taken when that file was still present. Still, it was created by `taskgraph init` and had issues.

**Comment by ahal** [comment]
I think Julien meant that this repo is not theTaskcluster repo, and any vulnerabilities here do not represent vulnerabilities in the CI system itself.

The scope of this repo is more or less limited to the decision task. That said I've been meaning to setup openssf here already and I don't see why we wouldn't want to fix these.

Also fwiw openssf was a requirement in the recent fxci-config rra I did, so seems prudent to get it going in general

**Comment by ahal** [comment]
I think there's some investigation required too. I think last time I looked openssf didn't know about pip-compile-multi, so thought our dependencies weren't locked and gave us a bogus score. This might block on switching to poetry
